sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/CreateMessageBatchOptions.cs

```diff
@@ -40,6 +43,30 @@ public class CreateMessageBatchOptions
             }
         }
 
+        /// <summary>
+        ///   The maximum size to allow for a single batch of messages, in number of items.
+        /// </summary>
+        ///
+        /// <value>
+        ///   The desired limit, in number of items, for the size of the associated service bus message batch.  If <c>null</c>,
+        ///   the maximum size allowed by the active transport will be used.
+        /// </value>
+        ///
+        internal int? MaxSizeInNumberOfItems
```

Comment:
At first, I had this public then I figured I'll start with making it internal first. Yet I guess this could be a nice add if wanted to the public API.